launcher_scripts/conf/config.yaml

```diff
@@ -2,7 +2,7 @@ defaults:
   - _self_
   - cluster: bcm  # Set to bcm for BCM and BCP clusters. Set to k8s for a k8s cluster.
   - data_curation: common_crawl/curate_common_crawl
-  - data_preparation: gpt3/download_gpt3_pile
```

Comment:
could you please return this line as a default?
and the same for `launcher_scripts/tests/unit_tests/config_tests/test_main_config.py`